test/e2e/snaps/ppom-malicious-erc20-transfer.spec.js

```diff
@@ -0,0 +1,210 @@
+const { strict: assert } = require('assert');
+const FixtureBuilder = require('../fixture-builder');
+const { mockServerJsonRpc } = require('../mock-server-json-rpc');
+
+const {
+  defaultGanacheOptions,
+  openDapp,
+  unlockWallet,
+  withFixtures,
+} = require('../helpers');
+
+const bannerAlertSelector = '[data-testid="security-provider-banner-alert"]';
+
+const selectedAddress = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
+const selectedAddressWithoutPrefix = '5cfe73b6021e818b776b421b1c4db2474086a7e1';
+const USDC_ADDRESS = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48';
```

Comment:
nit: we could move this inside the CONTRACT_ADDRESS object?

Reply:
updated [d0f19d4](https://github.com/MetaMask/metamask-extension/pull/21086/commits/d0f19d4361132baa2478f67d76409dacea1a1246)